dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityAdminGroupRestControllerIT.java

```diff
@@ -437,4 +457,376 @@ public void deleteCommunityAdminGroupNotFoundTest() throws Exception {
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADMIN, eperson);
+        Group group = GroupBuilder.createGroup(context).withName("testGroup").build();
+        configurationService.setProperty("core.authorization.community-admin.admin-group", false);
+        context.restoreAuthSystemState();
+
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(
+            post("/api/eperson/groups/" + adminGroup.getID() + "/subgroups")
+                .contentType(parseMediaType
+                    (org.springframework.data.rest.webmvc.RestMediaTypes
+                         .TEXT_URI_LIST_VALUE))
+                .content("https://localhost:8080/server/api/eperson/groups/" + group.getID()))
+            .andExpect(status().isForbidden());
+
+        token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/eperson/groups/" + adminGroup.getID() + "/subgroups"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.subgroups", Matchers.not(Matchers.hasItem(
+                            GroupMatcher.matchGroupWithName(group.getName())
+                        ))));
+
+        context.turnOffAuthorisationSystem();
+        configurationService.setProperty("core.authorization.community-admin.admin-group", true);
+        context.restoreAuthSystemState();
+
+    }
+
+    @Test
+    public void communityAdminRemoveChildGroupFromCommunityAdminGroupPropertySetToFalse() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+        Group adminGroup = communityService.createAdministrators(context, parentCommunity);
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADMIN, eperson);
+        Group group = GroupBuilder.createGroup(context).withName("testGroup").build();
+        context.restoreAuthSystemState();
+
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(
+            post("/api/eperson/groups/" + adminGroup.getID() + "/subgroups")
+                .contentType(parseMediaType
+                    (org.springframework.data.rest.webmvc.RestMediaTypes
+                         .TEXT_URI_LIST_VALUE))
+                .content("https://localhost:8080/server/api/eperson/groups/" + group.getID()));
+
+        getClient(token).perform(get("/api/eperson/groups/" + adminGroup.getID() + "/subgroups"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.subgroups", Matchers.hasItem(
+                            GroupMatcher.matchGroupWithName(group.getName())
+                        )));
+
+        context.turnOffAuthorisationSystem();
+        configurationService.setProperty("core.authorization.community-admin.admin-group", false);
+        context.restoreAuthSystemState();
```

Comment:
No need to turn off authorization to change a config value. remove context lines